lib/pbench/agent/tool_meister.py

```diff
@@ -855,9 +841,13 @@ def __init__(
             )
 
         # Name of the channel on which this Tool Meister instance will listen.
-        self._to_tms_channel = f"{self._channel_prefix}-{tm_channel_suffix_to_tms}"
+        self._to_tms_channel = (
+            f"{self._params.channel_prefix}-{tm_channel_suffix_to_tms}"
+        )
```

Comment:
Sigh; well, `black` clearly loves Webb's suggestion to encapsulate `_params`... 🍤

Reply:
Yeah...the problem is the leading underscores -- if we remove them, then I think Black will put this all on one line.  🤣